2-Authorization-I/1-call-graph/SPA/src/app/utils/claim-utils.ts

```diff
@@ -0,0 +1,194 @@
+/**
```

Comment:
preserve the same pattern in file names, so if you name this claim-utils.ts, the other can be storage-utils.ts etc.